x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_about_rule/schema.tsx

```diff
@@ -100,6 +100,17 @@ export const schema: FormSchema<AboutStepRule> = {
     ),
     labelAppend: OptionalFieldLabel,
   },
+  maxSignals: {
+    type: FIELD_TYPES.NUMBER,
+    helpText: i18n.translate(
+      'xpack.securitySolution.detectionEngine.createRule.stepAboutRule.fieldMaxSignalsHelpText',
+      {
+        defaultMessage:
+          'The maximum number of alerts a rule will create per execution. Defaults to 100.',
```

Comment:
@joepeeples Thoughts on this? Pretty straightforward language-wise but I'd like to get your opinion nonetheless

Reply:
Thanks for the ping, suggested edit below. Also, is the name "max signals" negotiable? I thought that "signals" was an outdated term that "alerts" replaced quite some time ago.

```suggestion
          'The maximum number of alerts the rule will create each time it runs. Default is 100.',
```

Reply:
Thanks! And yeah I left a [comment below](https://github.com/elastic/kibana/pull/179680#issuecomment-2030612049) with the same question. I'm happy to change the UI facing things to Max alerts or something similar if you don't think that would cause any confusion language-wise.